docs/data/material/components/snackbars/ToolpadNotifications.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import {
+  useNotifications,
+  NotificationsProvider,
+} from '@toolpad/core/useNotifications';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Switch from '@mui/material/Switch';
+
+function NotifyRadioButton() {
+  const notifications = useNotifications();
+  const [online, setOnline] = React.useState(true);
+  const prevOnline = React.useRef(online);
+  React.useEffect(() => {
+    if (prevOnline.current === online) {
+      return () => {};
+    }
+    prevOnline.current = online;
+
+    // preview-start
+    const key = online
+      ? notifications.show('You are now online', {
```

Comment:
Can we change the position of the notification? I almost didn't saw it.

Reply:
The only way I can see how is to do this sort of a scoped notification: https://mui.com/toolpad/core/react-use-notifications/#scoped-notifications

Reply:
Then it sounds like a missing API. Creating an issue for Toolapd could be enough.

Reply:
But the position is the same as the other demos. Isn't this just hard to see because of the color? Maybe it's a matter of matching the look to the other demos?

Reply:
Yes, to match the look we need to add  an `alert` key to the `slotProps` for `NotificationsProvider` so that we can pass in `variant: "filled"` to the `Alert` being used in case a `severity` prop is passed (which is the case in this demo). I can make an issue on Toolpad for this, let me know what you think

Reply:
> the position is the same as the other demos. Isn't this just hard to see because of the color

Right fair, the position is correct, it's the visual design that is broken. https://notistack.com/features/basic for error looks correct;